test/libsolidity/semanticTests/functionCall/mapping_array_internal_argument.sol

```diff
@@ -22,7 +22,7 @@ contract test {
 // set(uint8,uint8,uint8,uint8,uint8): 1, 21, 22, 42, 43 -> 0, 0, 0, 0
 // gas irOptimized: 110993
 // gas legacy: 111406
-// gas legacyOptimized: 107981
+// gas legacyOptimized: 109773
```

Comment:
Here is the diff: https://gist.github.com/hrkrshnn/e4c5b5830da1f66b885cd847f36e7643